packages/link/src/Link.js

```diff
@@ -57,6 +57,7 @@ const AvLink = ({ tag: Tag, href, target, children, onClick, loadApp, ...props }
       href={url}
       target={target}
       style={linkStyles}
+      className="link"
```

Comment:
This will be overwritten when someone passes in their own `className` prop. Is that ok with the styles or do we need to combine them?

Reply:
added.